src/components/APIDoc/Operation.tsx

```diff
@@ -29,32 +35,59 @@ export interface OperationProps {
   document: OpenAPIV3.Document;
 }
 
-export const Operation: React.FunctionComponent<OperationProps> = props => {
+export const Operation: React.FunctionComponent<OperationProps> = (props) => {
   const id = `operation-${props.verb}-${props.path}`;
   const [isExpanded, setExpanded] = useState(false);
-  const {operation, verb, path} = props;
+  const { operation, verb, path } = props;
 
-  return <AccordionItem>
-    <AccordionToggle
-      id={id}
-      isExpanded={isExpanded}
-      onClick={() => setExpanded(prev => !prev)}
-      className="pf-u-py-sm"
+  return (
+    <AccordionItem>
+      <AccordionToggle
+        id={id}
+        isExpanded={isExpanded}
+        onClick={() => setExpanded((prev) => !prev)}
+        className="pf-u-py-sm"
       >
-        {operation.summary && <span className="pf-u-font-weight-normal pf-u-color-100 pf-u-mr-lg">{operation.summary}<br /></span>}
-         <span className="pf-u-font-size-sm pf-u-font-weight-normal pf-u-color-200">{verb.toUpperCase()} {path}</span>
+        {operation.summary && (
+          <span className="pf-u-font-weight-normal pf-u-color-100 pf-u-mr-lg">
+            {operation.summary}
+            {operation.deprecated && (
+              <Label
+                color="red"
+                icon={<InfoCircleIcon />}
+                className="pf-u-ml-lg"
+              >
+                Deprecated
+              </Label>
```

Comment:
Looking good! I just had one suggestion. I think either `orange` or `gold` might look a little better here, since they are associated with a `warning` in patternfly: [Here](https://www.patternfly.org/design-foundations/colors/#primary-colors)

Reply:
Changed it to orange!